diff -Naur a/include/GLFW/glfw3native.h b/include/GLFW/glfw3native.h
--- a/include/GLFW/glfw3native.h	2026-09-08 17:31:46
+++ b/include/GLFW/glfw3native.h	2026-09-08 17:31:46
@@ -109,7 +109,7 @@
   #if defined(__OBJC__)
    #import <Cocoa/Cocoa.h>
   #else
-   #include <ApplicationServices/ApplicationServices.h>
+   //#include <ApplicationServices/ApplicationServices.h>
    #include <objc/objc.h>
   #endif
  #endif
@@ -268,7 +268,7 @@
  *
  *  @ingroup native
  */
-GLFWAPI CGDirectDisplayID glfwGetCocoaMonitor(GLFWmonitor* monitor);
+//GLFWAPI CGDirectDisplayID glfwGetCocoaMonitor(GLFWmonitor* monitor);
 
 /*! @brief Returns the `NSWindow` of the specified window.
  *
